.cirrus.yml

```diff
@@ -116,29 +211,89 @@ task:
   only_if: $CIRRUS_PR != ''
 
   container:
-    image: ponylang/ponyc-ci-x86-64-unknown-linux-gnu-builder:20191105
+    image: ponylang/ponyc-ci-x86-64-unknown-linux-gnu-builder:20191106
     cpu: 8
     memory: 24
 
   name: "PR: x86-64-unknown-linux-gnu"
 
+  libs_cache:
+    folder: build/libs
+    fingerprint_script: echo "`md5sum lib/CMakeLists.txt` glibc"
+    populate_script: make libs build_flags=-j8
+
+  configure_script:
+    - make configure
+  build_script:
+    - make build build_flags=-j8
+  test_script:
```

Comment:
This seems to be a duplicate of the task that starts on 232.

Reply:
O i c. we don't do debug and non-debug anymore. You can kill the debug ones.

Reply:
Actually I just re-added the debug builds because the debug build was broken and I thought it would be useful to check it.

Reply:
what was broken about it? it definitely adds a lot more time to PRs but perhaps with caching it won't?

Reply:
We recently added a bunch of bare `inline` functions to a header in libponyrt.  I learned something new yesterday: for unoptimized builds C99 evidently requires these functions to be declared in a `.c` file somewhere. So the build worked for release (where the functions were being inlined because optimization) but not in debug.
https://github.com/ponylang/ponyc/pull/3234/commits/ca1e2cbcd08a80d6443a6cb904f2ad73aafc0a05#diff-894a332d21d5ee3397dc394ab3dae5cc

Reply:
I think it would be worth leaving them in to catch things like that.  Perhaps we can turn off the test step and just compile them.